File: src/main/java/net/igneo/icv/entity/weapon/boostCharge/BoostChargeRenderer.java
package net.igneo.icv.entity.weapon.boostCharge;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class BoostChargeRenderer extends GeoEntityRenderer<BoostChargeEntity> {
    
    public BoostChargeRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new BoostChargeModel());
    }
}

File: src/main/java/net/igneo/icv/entity/weapon/boostCharge/BoostChargeRenderer.kt
package net.igneo.icv.entity.weapon.boostCharge

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class BoostChargeRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<BoostChargeEntity?>(renderManager, BoostChargeModel())
